code/datums/ai/basic_mobs/targetting_datums/basic_targetting_datum.dm

```diff
@@ -19,7 +19,7 @@
 	var/stat_attack = CONSCIOUS
 
 /datum/targetting_datum/basic/can_attack(mob/living/living_mob, atom/the_target)
-	if(isturf(the_target) || !the_target) // bail out on invalids
+	if(isturf(the_target) || !the_target || isitem(the_target.loc)) // bail out on invalids
```

Comment:
dont like specifically checking for items, can we do indestructible stuff instead?

Reply:
They cant hit items with melee or ranged attacks in the first place, checking for indestructibility would be pointless

Reply:
why, theres still indestructable structures/machines/vehicles you can hide in. and not being able to be hit with ranged/melee attacks seems like a targeting issue?

Reply:
what i meant was that they cant hit items at all and most items dont have any interaction for being shot or meleed on the floor (unlike carbons who can pick them up and all that) so checking for indestructibility would be pointless

also like the sole mob that i know of
that is held inside a holdable item is a
pAI

Reply:
such as in this case where this goofy ranged mob attempts to hit the pAI mob hiding inside the pAI card but cannot because their bullets cannot hit at all and causing a pile of 12000 bullets which would crash anyone nearby

![209937083-f9faedca-a73c-4b4a-9536-a037b44f53ac](https://user-images.githubusercontent.com/70376633/210048983-dc6055d0-3878-4f4d-9de3-bd2b6aaacd3e.png)

Reply:
items can be targeted by projectiles though, carbons can shoot them, thats how you deal with his grace, i dont see why we cant make mobs do that?
also i dont see how anything you said is relevant to checking for being indestructable. theres still structures people can hide in that are indestructable and would probably cause the same pile of bullets

Reply:
the issue here is that the syndie literally cannot kill the mob, cannot hit the pAI card (and even if it could it wouldnt do shit),  and will do this for several hours until de-aggroed by someone

also what indestructible structures

Reply:
> cannot hit the pAI card (and even if it could it wouldnt do shit)

yeah and thats cause the card is indestructible

Reply:
most items literally do not do shit when shot

Reply:
also making them able to hit the object is outside of the scope of this PR

Reply:
ok fuck it